src/components/RightSidebar/Participants/ParticipantsTab.vue

```diff
@@ -121,13 +124,20 @@ export default {
 		},
 	},
 
-	setup() {
+	setup(props) {
+		const { isActive } = toRefs(props)
 		const { sortParticipants } = useSortParticipants()
 		const isInCall = useIsInCall()
+		const {
+			participantsInitialised,
```

Comment:
Always false, as you don't fetch it from ParticipantsTab anymore, and don't change refs value
![image](https://github.com/nextcloud/spreed/assets/93392545/ffeb8d51-9e31-4933-8bca-1de4c7c9b303)

Should we move `participantsInitialised` to participantsStore as getter?
> [!TIP]
> Each conversation adds a user participant to the store, but it's a dummy object (not the complete, as we get from server). We could check in getter, if store has only this participant
https://github.com/nextcloud/spreed/blob/38d19fe4082b7cbadaff16af4b5b6a7c537834af/src/store/conversationsStore.js#L281